app/_src/gateway/plugin-development/wasm.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Create a Proxy-Wasm filter
+book: plugin_dev
+chapter: 11
+---
+
+## Your first Proxy-Wasm filter
+
+Filters can be written as a compilation target in almost any language that supports WebAssembly, but the easiest way to get started is by
```

Comment:
> Filters can be written as a compilation target in almost any language that supports WebAssembly

This first sentence in the documentation is not correct. "almost any language that supports WebAssembly," is not a reasonable expectation, in reality, many languages already support some form of Wasm compilation, but *very few* of them have a proxy-wasm SDK. Imo this needs to be reworded to clearly lay out expectations that filters can be written if:
1) the language supports a Wasm target (many languages fall into this category)
2) a proxy-wasm SDK is written (very few languages fall into this 2nd category).